mde-services/src/main/java/de/terrestris/mde/mde_backend/service/IsoGenerator.java

```diff
@@ -65,7 +70,7 @@ public static String replaceValues(String text) {
     return text;
   }
 
-  public File generateMetadata(String metadataId) throws XMLStreamException, IOException {
```

Comment:
Wouldn't it be usefull to keep the File generation in a seperate function and just reuse it in the validate function?

Reply:
Definitely. I'm in the process of implementing the publication, so I'll refactor this now.